.eslintrc.json

```diff
@@ -59,7 +59,7 @@
         "keyword-spacing": "error",
         "line-comment-position": "error",
         "linebreak-style": "error",
-        "max-depth": "error",
```

Comment:
any particular reason for this change?

Reply:
Nesting in ConversationManagersService was one too deep.

Reply:
oh okay. Just wanted to document it somewhere.